resources/dicts/patrols/general/training.json

```diff
@@ -11690,5 +11690,371 @@
+               "exp": 10,
+               "can_have_stat": ["p_l"],
+                "stat_trait": [ "childish", "playful" ],
+               "relationships": [
+                   {
+                       "cats_to": ["p_l"],
+                       "cats_from": ["app1"],
+           "mutual": true,
+                       "values": ["platonic", "comfort", "trust", "respect"],
+                       "amount": 15
+                   } ],
+               "weight": 20
+           },
+           {
+               "text": "s_c admits that sometimes {PRONOUN/s_c/subject} {VERB/s_c/feel/feels} the pressure of being loved and relied on by so many cats. If {PRONOUN/s_c/subject} knew {PRONOUN/s_c/subject} could protect {PRONOUN/s_c/poss} Clanmates, s_c would feel a lot better. p_l agrees to help, touched by {PRONOUN/s_c/poss} reasoning.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "charismatic", "loving", "loyal", "responsible", "compassionate" ],
+               "relationships": [
+                   {
+                       "cats_to": ["p_l"],
+                       "cats_from": ["app1"],
+           "mutual": true,
+                       "values": ["platonic", "comfort", "trust", "respect"],
+                       "amount": 15
+                   } ],
+               "weight": 20
+           },
+           {
+               "text": "p_l asks {PRONOUN/s_c/object} why {PRONOUN/s_c/subject} {VERB/s_c/want/wants} to learn such advanced battle moves. s_c looks at {PRONOUN/p_l/object} like that's a mouse-brained question - {PRONOUN/s_c/subject} {VERB/s_c/want/wants} to be the best, obviously! p_l purrs and agrees to show {PRONOUN/s_c/object} some moves.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "bold", "daring", "insecure", "confident", "arrogant", "ambitious" ],
+               "relationships": [
+                   {
+                       "cats_to": ["p_l"],
+                       "cats_from": ["app1"],
+           "mutual": true,
+                       "values": ["platonic", "comfort", "trust", "respect"],
+                       "amount": 15
+                   } ],
+               "weight": 20
+           },
+           {
+               "text": "s_c confesses {PRONOUN/s_c/poss} fear that one day, all of c_n will be in danger, and only {PRONOUN/s_c/subject} will be able to save them. p_l comforts s_c and assures {PRONOUN/s_c/object} that c_n will stand together against any threat. Feeling reassured, s_c asks if p_l will teach {PRONOUN/s_c/object} some moves anyway - for fun.",
+               "exp": 10,
+               "can_have_stat": ["app1"],
+                "stat_trait": [ "insecure", "nervous", "gloomy" ],
```

Comment:
ooh this feels like "loving" could also apply! in a "i feel deep love towards my clanmates and want to be able to protect them" way